website/config/custom.js

```diff
@@ -240,7 +240,7 @@ module.exports.custom = {
     '.github/ISSUE_TEMPLATE': ['mikermcneil', 'lukeheath', 'hollidayn'],
     '.github/ISSUE_TEMPLATE/bug-report.md': ['xpkoala','noahtalerman', 'lukeheath'],
     '.github/ISSUE_TEMPLATE/feature-request.md': ['xpkoala','noahtalerman', 'lukeheath'],
-    '.github/ISSUE_TEMPLATE/smoke-tests.md': ['xpkoala','lukeheath','noahtalerman', 'lukeheath'],
+    '.github/ISSUE_TEMPLATE/release-qa.md': ['xpkoala','lukeheath','noahtalerman', 'lukeheath'],
```

Comment:
up to @Sampfluger88 to verify this - I did not check the CODEOWNERS file to make sure it's not also in there

Reply:
All good! 

My understanding is DRI's conflicting with CODEOWNERS is where we run into problems. correct?
e.g. .github/ISSUE_TEMPLATE called out on line 240 here and in CODEOWNERS.

![image](https://github.com/fleetdm/fleet/assets/108141731/18b52ae5-7c85-45fc-9b5b-9a2689a4e216)